cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -188,15 +188,15 @@ def delete_org(self):
         p = sfdx("force:org:delete -p", self.username, "Deleting scratch org")
 
         stdout = []
-        for line in p.stdout_text:
+        for line in p.stderr_text:
             stdout.append(line)
```

Comment:
Let's please not take stderr and call it stdout

I want to make sure we're not suppressing useful information from sfdx about why the deletion failed. How sure are you that everything relevant is in stderr?